lib/spack/docs/getting_started.rst

```diff
@@ -866,11 +866,11 @@ Environment Modules
 """""""""""""""""""
 
 In order to use Spack's generated environment modules, you must have
-installed the *Environment Modules* package.  On many Linux
-distributions, this can be installed from the vendor's repository.
-For example: """yum install environment-modules``
-(Fedora/RHEL/CentOS).  If your Linux distribution does not have
-Environment Modules, you can get it with Spack:
+installed one of *Environment Modules* or *Lmod*.  On many Linux
+distributions, this can be installed from the vendor's repository.  For
+example: """yum install environment-modules`` (Fedora/RHEL/CentOS).  If
```

Comment:
I think I commented on this before, but is "yum install environment-moduels" supposed to be wrapped in double backticks or triple quotes? I assume double backticks.